Add missing MediaList subpages (#24386)
mdn/content: files/en-us/web/api/medialist/deletemedium/index.md

---
title: MediaList.deleteMedium()
slug: Web/API/MediaList/deleteMedium
page-type: web-api-instance-method
browser-compat: api.MediaList.deleteMedium 
---

{{APIRef("CSSOM")}}

> The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removed the media typquerye given in parameter from the list.

Reviewer: ```suggestion
The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removed the media type query given in parameter from the list.
```

Author: ```suggestion
The `deleteMedium()` method of the {{DOMxRef("MediaList")}} interface removes from this `MediaList` the given media query.
```

## Syntax

```js-nolint
deleteMedium(medium)
```

### Parameters

- `medium`
  - : A string containing the media query to remove from the list.

### Return value

None ([undefined](/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)).

### Exceptions

- `NotFoundError` {{domxref("DOMException")}}
  - : Thrown when the media query to remove is not in the list.

## Examples

> The following removes the media query `print` into the
> `MediaList` associated with the first stylesheet applied to the current document.

Reviewer: ```suggestion
The following removes the media query `print` from the
`MediaList` associated with the first stylesheet applied to the current document.
```
